tensorflow/python/compiler/tensorrt/test/binary_tensor_weight_broadcast_test.py

```diff
@@ -60,7 +60,10 @@ def GetParams(self):
 
   def ExpectedEnginesToBuild(self, run_params):
     """Return the expected engines to build."""
-    return ["TRTEngineOp_%d" % i for i in range(16)]
+    expected_engines = ["TRTEngineOp_%d" % i for i in range(16)]
+    if run_params.dynamic_shape:
+      expected_engines.append("TRTEngineOp_16")
+    return expected_engines
```

Comment:
NIT: can be refined to

num_engines = 17 if .. else 16
expected_engines = ["TRTEngineOp_%d" % i for i in range(num_engines)]